src/fu-util-common.c

```diff
@@ -460,3 +460,38 @@ fu_util_setup_networking (GError **error)
 	soup_session_remove_feature_by_type (session, SOUP_TYPE_CONTENT_DECODER);
 	return g_steal_pointer (&session);
 }
+
+static const gchar *
+fu_util_release_scope_localized (const gchar *scope)
+{
+	if (g_strcmp0 (scope, "device") == 0) {
+		/* TRANSLATORS: a specific part of hardware */
+		return _("Device Firmware");
+	}
+	if (g_strcmp0 (scope, "system") == 0) {
+		/* TRANSLATORS: the entire system, e.g. all internal devices */
+		return _("System Firmware");
+	}
+	if (g_strcmp0 (scope, "ec") == 0) {
+		/* TRANSLATORS: the EC is typically the keyboard controller chip */
+		return _("Embedded Controller Firmware");
+	}
+	if (g_strcmp0 (scope, "me") == 0) {
+		/* TRANSLATORS: ME stands for Management Engine, the Intel AMT thing */
+		return _("ME Firmware");
+	}
+	/* TRANSLATORS: this is the fallback where we don't know if the release
+	 * is updating the system, the device, or a device class */
+	return _("Firmware");
+}
+
+gchar *
+fu_util_release_get_name (FwupdRelease *release)
+{
+	const gchar *scope = fwupd_release_get_scope (release);
+	if (scope == NULL)
+		return g_strdup (fwupd_release_get_name (release));
+	return g_strdup_printf ("%s %s",
+				fwupd_release_get_name (release),
```

Comment:
Is it true that all languages will want to place this after the device name itself?

Reply:
That's true indeed. Willfix.

Reply:
Fixed.